lib/Mad/Mapper.pm

```diff
@@ -287,16 +299,20 @@ sub _delete {
     );
   }
   else {
-    $self->db->query(@sst);
+    $self->db->query(@sql);
     $self->in_storage(0);
   }
 }
 
-sub _delete_sst {
+sub _delete_sql {
```

Comment:
Oh! All the _delete_sst, _find_sst, _insert_sst and _update_sst need to be deprecated as well, since they are documented as "public" methods for developers.

Maybe something like this:

```
for (qw(_delete, _find, _insert _update)) {
  my ($old, $new) = ("${_}_sst", "${_}_sql");
  *$old = sub {
    Mojo::Util::deprecated("$old() is deprecated in favor of new()");
    shift->$new(@_);
  };
}
```